indexing/how-search-engines-index-content-better-discoverability.md

```diff
@@ -1,17 +1,22 @@
 ---
-layout: post
+layout: page
 title: How to get search engines to index the right content for better discoverability
 crumbname: Better Discoverability
 category: manual
```

Comment:
Let's do a bulk update to change `category: manual` into `category: admin-center`. OR update the term AND convert to a tag.

Reply:
Bulk update seems easier for now, are we linking out or using categories currently?